packages/dx-react-grid/docs/reference/grid.md

```diff
@@ -53,6 +53,15 @@ Field | Type | Description
 ------|------|------------
 children? | ReactElement | A React element to be placed in the footer.
 
+## Plugin Components
+
+Name | Properties | Description
+-----|------------|------------
+Grid.Root | [GridRootProps](#gridrootprops) | A component that renders tables body.
```

Comment:
A component that renders tables [root layout](https://github.com/DevExpress/devextreme-reactive/blame/master/packages/dx-react-grid/docs/reference/grid.md#L15)?